src/hooks/useComments.ts

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { Comment } from '../types/Comment';
+import { addComment, deleteComment } from '../api/comments';
+import { getCommentsByPostId } from '../api/posts';
+
+export const useComments = (postId: number) => {
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [isLoading, setLoading] = useState(false);
+  const [isError, setError] = useState('');
```

Comment:
```suggestion
  const [error, setError] = useState('');
```
`is` prefix should be used only for boolean values